filter_plugins/utils.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import bcrypt
```

Comment:
Yes, it needs to be installed.  Add a note to readme for `pip install django[bcrypt]`?